mainnet-canisters.bzl

```diff
@@ -23,9 +23,9 @@ CANISTER_NAME_TO_WASM_METADATA = {
     "sns_ledger": ("3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d", "e8942f56f9439b89b13bd8037f357126e24f1e7932cf03018243347505959fd4"),
     "sns_archive": ("3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d", "5c595c2adc7f6d9971298fee2fa666929711e73341192ab70804c783a0eee03f"),
     "ck_btc_index": ("a3831c87440df4821b435050c8a8fcb3745d86f6", "cac207cf438df8c9fba46d4445c097f05fd8228a1eeacfe0536b7e9ddefc5f1c"),
-    "ck_btc_ledger": ("a3831c87440df4821b435050c8a8fcb3745d86f6", "4264ce2952c4e9ff802d81a11519d5e3ffdaed4215d5831a6634e59efd72f7d8"),
+    "ck_btc_ledger": ("d323465e02b84a0bc3b8c2c6fd362f6072f1a3f2", "cb5fc1cd94cb75791e8c01be3116122d779ef61c6eab41f10b60f6e79fe9f0e9"),
```

Comment:
Changes in this file should be reverted when mainnet canisters are updated to V1.

Reply:
Reverted!